tsfile/src/main/java/org/apache/iotdb/tsfile/file/header/ChunkHeader.java

```diff
@@ -76,20 +75,8 @@ public static int getSerializedSize(String measurementID) {
    * @param markerRead Whether the marker of the CHUNK_HEADER has been read
    */
   public static ChunkHeader deserializeFrom(InputStream inputStream, boolean markerRead) throws IOException {
-    if (!markerRead) {
-      byte marker = (byte) inputStream.read();
-      if (marker != MetaMarker.CHUNK_HEADER) {
-        MetaMarker.handleUnexpectedMarker(marker);
-      }
-    }
-
-    String measurementID = ReadWriteIOUtils.readString(inputStream);
-    int dataSize = ReadWriteIOUtils.readInt(inputStream);
-    TSDataType dataType = TSDataType.deserialize(ReadWriteIOUtils.readShort(inputStream));
-    int numOfPages = ReadWriteIOUtils.readInt(inputStream);
-    CompressionType type = ReadWriteIOUtils.readCompressionType(inputStream);
-    TSEncoding encoding = ReadWriteIOUtils.readEncoding(inputStream);
-    return new ChunkHeader(measurementID, dataSize, dataType, type, encoding, numOfPages);
+    ChunkHeaderSerializer serializer = new ChunkHeaderSerializer(); 
+    return serializer.deserializeFrom(inputStream, markerRead);
```

Comment:
`return new ChunkHeaderSerializer().deserializeFrom(inputStream, markerRead);`. (avoid to generate a temporary object reference)

Reply:
Why not make the deserializeFrom a static method?